### ISSUE TITLE
IformFile in the action controller takes the value as null, value is pass to the action controller using ajax request

### ISSUE THREAD

**Issue by sarbeshbanskota123** [domaindrivendev/Swashbuckle.AspNetCore#1456]
am trying to get the value passed from the ajax request into one of the method in the controller but IFormFile files is null every time. It is hitting the right method. In the console network I can see the file but the receiver is getting the value as null. Here is my ajax request :

 uploadFile: function (field, value) 
{
    var me = this;
    var view = me.getView();
    var fileuploadControl = me.lookupReference('ImportFile');
    var file = fileuploadControl.fileInputEl.el.dom.files[0];   
    var param = new FormData();
    param.append('files', file);
    var ajax = Ext.Ajax.request(
  {

        url: './../XYController/ImportCSVFile',
                data: param,
                method: 'POST',
                contentType: "application/json; charset=utf-8",
                dataType: "json",
                headers: { 'accept': '*/*' },
                processData: true,
                success: function (response, options) {
                    var result = JSON.parse(response.responseText);
                    if (mask) {
                        mask.destroy();
                    }   
                    Ext.Msg.alert("File Upload Successful");
   }

});
},

And this is my Action Controller :

[HttpPost]
[Route("XYController/ImportCSVFile")]
public IActionResult ImportCSVFile(IFormFile files)
{
  if(files!=null)
   {
    //do something
   }
}

**Comment by domaindrivendev** [comment]
if I’m understanding correctly, you’re having a problem uploading a file to your ASP.NET API. This seems like a general ASP.NET question - I’m not seeing anything specific to Swashbuckle in your issue. Am I missing something?

**Comment by domaindrivendev** [comment]
Closing due to inactivity
